examples/lib/lib_test.go

```diff
@@ -28,3 +28,9 @@ func TestMeaning(t *testing.T) {
 func TestAddTwoNumbers(t *testing.T) {
 	AddTwoNumbers()
 }
+
+func TestBuildTime(t *testing.T) {
+	if buildTime == "" {
```

Comment:
It is better if we can keep this test passing with just "go test" command.
So could you accept the expected value as a flag?

```go
var (
        buildTimeWant = flag.String("lib_test.buildtime", "", "expected value in TestBuildTime")
)
...
func TestBuildTime(t *testing.T) {
        if got, want := buildTime, *buildTimeWant; got != want {
               t.Errorf("buildTime = %q; want %q", got, want)
        }
}
```

You can specify the flag with `args` attribute of `go_test` rule.